sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ParameterAssignedTo.cs

```diff
@@ -222,9 +222,19 @@ public string foo(string x)
         {
             if (x == null)
             {
-                x = ""; // Noncompliant FP
+                x = ""; // Noncompliant FP (https://github.com/SonarSource/sonar-dotnet/issues/2603)
```

Comment:
```suggestion
                x = ""; // Noncompliant FP (https://github.com/SonarSource/sonar-dotnet/issues/2555)
```

Reply:
I wanted #2603 since it's a duplicate of #2555 which specifies exactly the above scenario

Reply:
ok